project/mkdocs.yml.jinja

```diff
@@ -3,6 +3,7 @@ site_description: "[[ project_description ]]"
 site_url: "https://[[ repository_namespace ]].github.io/[[ repository_name ]]"
 repo_url: "https://github.com/[[ repository_namespace ]]/[[ repository_name ]]"
 repo_name: "[[ repository_namespace ]]/[[ repository_name ]]"
+site_dir: '[% if repository_provider == "gitlab.com" %]public[% elif repository_provider == "github.com" %]site[% endif %]'
```

Comment:
I believe this is the right format for this but if not feel free to edit it to be formatted properly for the jinja template. I used the readme.MD.jinja template as a reference.